### DISCUSSION TITLE
Okta integration with Timesketch

### DISCUSSION THREAD

**Discussion by GT-SteveG** [google/timesketch#2915] [Q&A, quick solutions, support]
Has anybody been able to get this working? With the settings below, Okta takes me to my site but doesn't sign in or give me the option to sign in. It essentially just seems to route to my website, nothing more. I know its something I'm doing wrong, so 
apologies in advance, this is a little out of my comfort zone. 

I've configured Okta to use OIDC, with the following settings:

_Sign-in redirect URIs https://mywebsite.com.com/auth/oidc/callback
Sign-out redirect URIs https://mywebsite.com.com
Login initiated by Either Okta or App
Initiate login URI https://mywebsite.com.com_

Then edited the timesketch.inf as below.......

Single Sign On (SSO) configuration.

Your web server can handle authentication for you by setting a environment
variable when the user is successfully authenticated. The standard environment
variable is REMOTE_USER and this is the default, but if your SSO system uses
another name you can configure that here.

SSO_ENABLED = True
SSO_USER_ENV_VARIABLE = 'REMOTE_USER'

GOOGLE_OIDC_ENABLED = True

GOOGLE_OIDC_AUTH_URL = 'https://mysite.okta.com/oauth2/v1/authorize'
GOOGLE_OIDC_DISCOVERY_URL = 'https://mysite.okta.com/.well-known/openid-configuration'
GOOGLE_OIDC_ALGORITHM = 'RS256'

GOOGLE_OIDC_CLIENT_ID = 'REDACTED'
GOOGLE_OIDC_CLIENT_SECRET = 'REDACTED'

**Comment by kyt999** [comment]
Hey @GT-SteveG did you ever get this working?
